packages/framework-provider-local/src/library/api-adapter.ts

```diff
@@ -3,11 +3,16 @@ import { httpStatusCodeFor, toClassTitle } from '@boostercloud/framework-types'
 export type APIResult =
   | { status: 'success'; result: unknown }
   | { status: 'failure'; code: number; title: string; reason: string }
+  | { status: 'success'; headers: unknown; result: unknown }
 
-export async function requestSucceeded(body?: any): Promise<APIResult> {
+export async function requestSucceeded(body?: any, headers?: Record<string, string>): Promise<APIResult> {
   return {
     status: 'success',
     result: body,
+    headers: {
+      'Access-Control-Allow-Origin': '*',
+      ...headers,
+    },
```

Comment:
I think that it's better if we set any default headers in the file `booster-graphql-dispatcher.ts` when the `GraphQLResolverContext` object is created, I'll leave another comment there.